src/main/webapp/app/shared/util/i18n-role.pipe.ts

```diff
@@ -0,0 +1,19 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { RoleUtil } from 'app/shared/util/role.util';
+
+/**
+ * Resolves a role by the provided `roleName` and returns its translated text.
+ */
+@Pipe({ name: 'i18nRolePipe' })
```

Comment:
why didn't you just use the jhipster `translate` pipe?

Reply:
Wir bekommen die id der Rolle (name). Die id alleine reicht aber nicht fürs i18n deshalb ist im model noch ein i18nKey property drauf.

Reply:
ja, aber die wäre ja eigentlich schon eindeutig.. also so auch ok, dann könnten aber die `role.constants.ts` entfernt werden, nicht?

Reply:
Ich verstehe nicht was du meinst. Die Namen der Rollen sind eindeutig ja, z.b. ADMIN, SECRETARY etc. Aber der "Pfad" für den i18n key ist ja nicht dieser.

Reply:
> dann könnten aber die role.constants.ts entfernt werden, nicht?
Das eine hat ja irgendwie nichts mit dem anderen zu tun? Die Konstanten für die Rollen brauchen wir ja sowieso. Dort ist einfach noch die Aufschlüsselung nach dem i18nKey drin.